presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java

```diff
@@ -334,7 +334,7 @@ protected void ensureHashTableLoaded()
 
             int[] offsets = getOffsets();
```

Comment:
why do we have `offsets` variable instead of accessing `this.offsets` directly?

Reply:
Removed. It was moved from AbstractMapBlock and I forgot to remove the line.